client/cz-cardview.html

```diff
@@ -152,6 +152,7 @@
       },
 
       attached: function() {
+        registerSource('cz-config', 'cardview-heading', cvHeading => this.set('cardview-heading', cvHeading));
```

Comment:
Add a default value (it can be something boring) so configs that don't specify a heading have something.